api-dummy/public/schemas/libero/article/body.rng

```diff
@@ -1,20 +1,32 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
-<grammar xmlns="http://relaxng.org/ns/structure/1.0" ns="http://libero.pub">
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+    xmlns:a="http://relaxng.org/ns/compatibility/annotations/1.0">
 
     <include href="../core.rng"/>
 
     <start>
+        <ref name="libero.body"/>
+    </start>
 
-        <element name="body">
-            <ref name="libero.body.content"/>
-        </element>
+    <div>
+        <a:documentation>body element</a:documentation>
 
-    </start>
+        <define name="libero.body">
+            <element name="body">
+                <ref name="libero.body.attributes"/>
+                <ref name="libero.body.content"/>
+            </element>
+        </define>
+
+        <define name="libero.body.attributes">
+            <ref name="libero.attributes.lang"/>
+        </define>
+
+        <define name="libero.body.content">
```

Comment:
`attributes` is clear enough but is `content` a standard word?

Reply:
don't want to overload it with, for example, the more specific `content` we use for various items like blog articles and such

Reply:
Borrowed from https://www.w3.org/MarkUp/2010/xhtml-m12n-relaxng-20101216/#s_relaxngrules_groups